end_to_end/test_decode_checkpoint.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+set -ue #x for debugging
+
+helpFunction()
```

Comment:
Is this being run as an integ test somewhere? It would probably be best to run these directly on the github runner?